test/server/graphql/v2/mutation/AccountMutations.test.ts

```diff
@@ -446,4 +449,135 @@ describe('server/graphql/v2/mutation/AccountMutations', () => {
       expect(collective.data.policies).to.be.empty;
     });
   });
+
+  describe('sendMessage', () => {
+    const sendMessageMutation = gqlV2/* GraphQL */ `
+      mutation SendMessage($account: AccountReferenceInput!, $message: NonEmptyString!, $subject: String) {
+        sendMessage(account: $account, message: $message, subject: $subject) {
+          success
+        }
+      }
+    `;
+
+    const message = 'Hello collective, I am reaching out to you for testing purposes.';
+
+    let sandbox, sendEmailSpy, collectiveWithContact, collectiveWithoutContact;
+
+    before(async () => {
+      sandbox = createSandbox();
+      collectiveWithContact = await fakeCollective({ admin: adminUser, settings: { features: { contactForm: true } } });
+      collectiveWithoutContact = await fakeCollective({
+        admin: adminUser,
+        settings: { features: { contactForm: false } },
+      });
+
+      sendEmailSpy = sandbox.spy(emailLib, 'sendMessage');
+    });
+
+    after(() => sandbox.restore());
+
+    afterEach(() => {
+      sendEmailSpy.resetHistory();
+    });
+
+    it('sends the message by email', async () => {
+      const result = await graphqlQueryV2(
+        sendMessageMutation,
+        {
+          account: { id: idEncode(collectiveWithContact.id, 'account') },
+          message,
+          subject: 'Testing',
+        },
+        randomUser,
+      );
+
+      expect(result.errors).to.not.exist;
+      expect(result.data.sendMessage.success).to.equal(true);
+
+      await waitForCondition(() => sendEmailSpy.callCount === 1);
+      expect(sendEmailSpy.callCount).to.equal(1);
+      expect(sendEmailSpy.args[0][1]).to.equal(
+        `New message from ${randomUser.collective.name} on Open Collective: Testing`,
+      );
+      expect(sendEmailSpy.args[0][2]).to.include(message);
+    });
+
+    it('cannot inject code in the email (XSS)', async () => {
+      const code = '<script>console.log("XSS")</script>';
+      const result = await graphqlQueryV2(
+        sendMessageMutation,
+        {
+          account: { id: idEncode(collectiveWithContact.id, 'account') },
+          message: message + code,
+          subject: 'Testing',
+        },
+        randomUser,
+      );
+
+      expect(result.errors).to.not.exist;
+      expect(result.data.sendMessage.success).to.equal(true);
+
+      await waitForCondition(() => sendEmailSpy.callCount === 1);
+      expect(sendEmailSpy.callCount).to.equal(1);
+      expect(sendEmailSpy.args[0][1]).to.equal(
+        `New message from ${randomUser.collective.name} on Open Collective: Testing`,
+      );
+      expect(sendEmailSpy.args[0][2]).to.include(message);
+      expect(sendEmailSpy.args[0][2]).to.not.include(code);
```

Comment:
Here we don't really guarantee that the injection did not happen. What if the code was simply turned into:

```html
<script>console.log('XSS')</script>
```

(simple quotes)

Would be great to test the exact message:

```suggestion
      expect(sendEmailSpy.args[0][2]).to.eq(`...`);
```